packages/core/src/components/Dialog/__stories__/Dialog.stories.tsx

```diff
@@ -538,14 +545,23 @@ export const ControlledDialog = {
         content={
           <DialogContentContainer>
             <DialogContentContainer>
-              <Button kind={Button.kinds.SECONDARY} onClick={() => setIsOpen(false)}>
+              <Button
+                kind={Button.kinds.SECONDARY}
+                // @ts-ignore
```

Comment:
Something is wrong with the useSwitch hook, it's also not typed so it's a bit tricky now